Pipfile

```diff
@@ -26,4 +26,4 @@ whitenoise = "*"
 [dev-packages]
 
 [requires]
-python_version = "3"
+python_version = "3.7"
```

Comment:
This may cause compatibility issues with the rest of us. Was it absolutely necessary to get things working your end?

Reply:
Ouu I'll change that - I was experimenting earlier and forgot to take it back to 3.